wiki/Beatmap_Editor/Timelines/en.md

```diff
@@ -1,137 +1,135 @@
+![Song's Timeline](/wiki/shared/BE_STL.jpg "Song's Timeline")
 
-At **centre**, it shows the **timeline with markings and the compulsory music player buttons.** For "Test" button, your beatmap will be saved to test the beatmap starting on current timestamp.
+The song player is visible in any part of the beatmap editor.
 
-### Colour markers
+On the left, there is the timestamp in milliseconds and the song duration percentage.
+The percentage may display `intro` or `outro` if there is storyboarding before or after the song.
 
-| Colour | Description |
-| ------ | ----------- |
-| Bright White Long	| Current test section |
-| Yellow Long	| Preview point. |
-| Yellow up	| Start of Drain time. |
-| Green up | Inherited Points. (See [Timing Setup](/wiki/Beatmap_Editor/Timing)) |
-| Red up | Timing Points. (See Timing Setup) |
-| Blue Down |	Bookmark |
+In the centre, it shows the timeline with markings and the compulsory music player buttons.
+The `Test` button will save your beatmap then will start play-testing it starting from the current timestamp.
 
-### Colour Highlights
+The timeline itself uses a few tick marks that have various meanings.
 
 | Colour | Description |
 | ------ | ----------- |
-| Grey | Break Time |
-| Orange | Kiai Time |
-
-### Bookmarking command
-
-| Shortcut | Description |
-| -------- | ----------- |
-| Ctrl + B | Add bookmark at current location. |
-| Ctrl + Shift + B | Remove bookmark at current location. |
-| Ctrl + Right Arrow | Next bookmark. |
-| Ctrl + Left Arrow	| Previous bookmark. |
+| White (long) | Current timestamp |
+| Yellow (long tick) | Preview point |
+| Yellow (up tick) | Start of drain time |
+| Green (up tick) | Inherited points (See [Timing](/wiki/Timing)) |
+| Red (up tick) | Timing points (See [Timing](/wiki/Timing)) |
+| Blue (down tick) | Bookmarks |
+| Grey (highlight) | Break time |
+| Orange (highlight) | Kiai time |
 
-At **right**, you can adjust the **Playback Rate** by **per quarter value.**
+On the right, you can adjust the playback rate from `100%`, to `75%`, `50%`, or `25%`.
 
-Compose
-------------
+## Hit Objects
 
-### osu!
+There are two different kinds of hit objects timelines depending on which game mode the mapper is using.
 
-*Location: Top @ [Compose](/wiki/Beatmap_Editor/Compose) tab in any modes other than osu!mania only.*
+### osu!, osu!taiko, and osu!catch
 
-![Hit Objects Timeline](/wiki/shared/BE_NTL.jpg "Hit Objects Timeline")
+![Hit Objects Timeline in osu!, osu!taiko, and osu!catch](/wiki/shared/BE_NTL.jpg "This shows the timeline for hit objects with respect to the beat snap divisor and timestamp.")
 
-**Shows the timeline for hitobjects with respect to [Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) and timestamp.**
-
-#### Hit Object Timeline
+In [compose](/wiki/Compose) mode, this timeline is underneath the `Compose` tab for any game mode except for [osu!mania](wiki/osu!mania).
```

Comment:
The first compose is talking about compose mode, not the article itself.
I will lowercase the c in compose (link) tho.